apps/field.go

```diff
@@ -3,39 +3,71 @@ package apps
 type FieldType string
 
 const (
-	FieldTypeText          = FieldType("text")
-	FieldTypeStaticSelect  = FieldType("static_select")
-	FieldTypeDynamicSelect = FieldType("dynamic_select")
-	FieldTypeBool          = FieldType("bool")
-	FieldTypeUser          = FieldType("user")
-	FieldTypeChannel       = FieldType("channel")
+	// Text field. "subtype":"textarea" for multi-line text input (Modal only?).
```

Comment:
There are more subtypes for modals (that we could easily implement in commands): number, password, url, email and tel (only for mobile and only useful in modals).

Reply:
create https://mattermost.atlassian.net/browse/MM-33909